lib/triggers/stream.js

```diff
@@ -0,0 +1,69 @@
+const elasticio = require('elasticio-node');
+const messages = elasticio.messages;
+const debug = require('debug')('salesforce:stream');
+const jsforce = require('jsforce');
+const MetaLoader = require('../helpers/metaLoader');
+const SALESFORCE_API_VERSION = '39.0';
+
+let conn;
+
+/**
+ * This method will be called from elastic.io platform providing following data
+ *
+ * @param msg incoming message object that contains ``body`` with payload
+ * @param cfg configuration that is account information and configuration field values
+ */
+function processTrigger(msg, cfg) {
+  const emitError = e => {
+    console.log('Oops! Error occurred');
+    console.log(e);
```

Comment:
Can these be put in the same `console.log` statement?  The platform has a problem of not ordering logs especially when the timestamps have the same ms level granularity.